js-exercises/animal-sort/animalSort.js

```diff
@@ -0,0 +1,12 @@
+const animalSort = animals => {
+  return animals.sort((currentObject, nextObject) => {
+    if (currentObject.numberOfLegs - nextObject.numberOfLegs === 0) {
+      const currentName = currentObject.name.toLowerCase();
+      const nextName = nextObject.name.toLowerCase();
+      return currentName > nextName ? 1 : -1;
+    }
+    return currentObject.numberOfLegs - nextObject.numberOfLegs;
+  });
```

Comment:
A separate comapre function can be written and passed as an argument instead of writing the whole method inside the sort. It'll improve code readability, modularity.